contrib/arch/d1x-rebirth/PKGBUILD

```diff
@@ -1,36 +1,44 @@
-# Contributor : zico <zico@dxx-rebirth.com>
+# Contributor: zico <zico@dxx-rebirth.com>
+# Contributor: Dominic Radermacher <dominic.radermacher@gmail.com>
+# Maintainer: SanskritFritz (gmail)
 
 pkgname=d1x-rebirth
-pkgver=0.61
-pkgrel=1
-pkgdesc="An enhanced engine to play with Descent1 datas"
-arch=('i686' 'x86_64')
+pkgver=20191121
+pkgrel=2
+pkgdesc="An enhanced engine to play with Descent1 data."
+arch=('x86_64')
 url="http://www.dxx-rebirth.com/"
 license=('custom:D1x' 'LGPL' 'custom:as-is')
 depends=('sdl' 'sdl_mixer' 'mesa' 'physfs')
-makedepends=('scons')
-install=$pkgname.install
-source=(git+https://github.com/dxx-rebirth/dxx-rebirth.git $pkgname.desktop $pkgname.install https://www.dxx-rebirth.com/download/dxx/res/d1xr-hires.dxa)
-noextract=(d1xr-hires.dxa d1xr-sc55-music.dxa)
-md5sums=('SKIP'
-         'ded1f8c2deb3825134e1a984985684cb'
-         '78c79b2b82175ebd4073edf257a9bd68'
-         '7d978ce3875995e54cbf426167e88b94') 
+makedepends=('scons' 'unzip')
+install="$pkgname.install"
+source=("https://www.dxx-rebirth.com/download/dxx/rebirth/dxx-rebirth_$pkgver-src.tar.xz"
+        "http://www.dxx-rebirth.com/download/dxx/res/d1xr-hires.dxa"
```

Comment:
These should be `https`.

Reply:
Yes, you're right.

Reply:
If Arch has dropped i686 support, I'm fine following their lead for the PKGBUILD file.  Leave this line as you have it.